process/testing/winfv/capz/install-calico.sh

```diff
@@ -66,9 +66,8 @@ if [ ${PRODUCT} == 'calient' ]; then
     ${KCAPZ} create -f ./EE/storage-class-azure-file.yaml
     ${KCAPZ} create -f ./EE/persistent-volume.yaml
 fi
-# Install Calico on Linux nodes
-curl -sSf -L --retry 5 ${RELEASE_BASE_URL}/manifests/tigera-operator.yaml -o tigera-operator.yaml
-${KCAPZ} create -f ./tigera-operator.yaml
+# Install Calico on Linux nodes using local manifests
+${KCAPZ} create -f ../../../../manifests/tigera-operator.yaml
```

Comment:
@coutinhop does this change looks good to you? Is there any better way to refer to the local manifest?

Reply:
Since this script is not always called from its own dir,  something like this would make it more robust (we've used it elsewhere):
```suggestion
SCRIPT_CURRENT_DIR="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 && pwd -P )"
${KCAPZ} create -f ${SCRIPT_CURRENT_DIR}../../../../manifests/tigera-operator.yaml
```
Though I am currently working on fixing this test and may do it myself in a separate PR if you'd prefer.

Reply:
Thanks, applied the suggestion.